ui/src/app/core/settings.service.ts

```diff
@@ -86,10 +103,16 @@ export class SettingsService {
   }
 
   private createEmptySettingsForProject(projectId: string): ProjectSettings {
+    const capabilities = this.capabilitiesService.getCapabilitiesSynchronous();
+    let hideArchivedValue;
+    if (capabilities.queryExtensions) {
+      hideArchivedValue = capabilities.queryExtensions.includes('hideArchived');
```

Comment:
So, just to be clear, the default behavior for a project it to hide archived jobs? And you'd have to explicitly set it to **not** do that.

Reply:
If it's in the query extensions, yes.